openwisp_controller/config/base/vpn.py

```diff
@@ -141,6 +149,14 @@ def get_context(self):
             c.update([('cert', self.cert.certificate), ('key', self.cert.private_key)])
         if self.dh:
             c.update([('dh', self.dh)])
+        # NOTE: Is this required?
```

Comment:
it can be useful and is also consistent (why not be able to use the subnet variables in the config? makes sense to me)